### PR TITLE
fix(DatePicker): Remove the shortcut to open the `HelperInformation` component.

### PR DESCRIPTION
**Pull request by nikitaorliak-cengage**
Issue: #1396

## What I did

Change the shortcut on the DatePicker to open `HelperInformation` from `?` to `Ctrl+Shift+?`. Add a tooltip for it. Write test. Change localization.

## Screenshots:

image
image

## Checklist 
- [x] changeset has been added
- [x] Pull request description is descriptive
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by silvalaura** [comment]
The panel opens with the new command on Safari and Firefox, but not on Chrome 🤔

**Comment by silvalaura** [comment]
> > The panel opens with the new command on Safari and Firefox, but not on Chrome 🤔
> 
> Sounds weird. I've tried using different keyboards and languages ​​in Chrome and Chromium and it works for me. Could you please double-check it?

I confirmed that on my PC, it works for all browsers (Chrome, Firefox and Edge). On my Mac, it does not work on Chrome but does on Firefox and Safari :/ @orion-cengage or @chris-cedrone-cengage , can you guys check if this works on your Macs?

File: packages/react-magma-dom/src/i18n/default.ts
Comment (silvalaura) [inline packages/react-magma-dom/src/i18n/default.ts:147]: changing this would be a breaking change for our adopters if they are already using `questionMark`, so maybe we should leave it as `questionMark` even if it's not 100% accurate

File: packages/react-magma-dom/src/components/DatePicker/CalendarMonth.tsx
Comment (silvalaura) [inline packages/react-magma-dom/src/components/DatePicker/CalendarMonth.tsx:198]: @orion-cengage Do you have a preference between lowercase or uppercase for the tooltip?

File: .changeset/a11y-datePicker-shortcut.md
Comment (nikitaorliak-cengage) [reply .changeset/a11y-datePicker-shortcut.md]: I changed docs in `Internationalization` section a bit, that's why I thought that it should be `docs`. Got it, thanks!

File: packages/react-magma-dom/src/i18n/default.ts
Comment (nikitaorliak-cengage) [reply packages/react-magma-dom/src/i18n/default.ts:147]: Oh, okay. I thought I'd better clarify that and change from `questionMark` to `ctrlShiftAndQuestionMark` everywhere in the code. Changed it back to `questionMark` :)

File: packages/react-magma-dom/src/components/DatePicker/CalendarMonth.tsx
Comment (nikitaorliak-cengage) [reply packages/react-magma-dom/src/components/DatePicker/CalendarMonth.tsx:198]: I used lowercase because saw them in the issue:

https://github.com/cengage/react-magma/issues/1396

image

Comment (orion-cengage) [reply packages/react-magma-dom/src/components/DatePicker/CalendarMonth.tsx:198]: Lower case is great

**Comment by orion-cengage** [comment]
> > > The panel opens with the new command on Safari and Firefox, but not on Chrome 🤔
> > 
> > 
> > Sounds weird. I've tried using different keyboards and languages ​​in Chrome and Chromium and it works for me. Could you please double-check it?
> 
> I confirmed that on my PC, it works for all browsers (Chrome, Firefox and Edge). On my Mac, it does not work on Chrome but does on Firefox and Safari :/ @orion-cengage or @chris-cedrone-cengage , can you guys check if this works on your Macs?

Nope, the new keyboard command is not working for me in Chrome on my Mac, but it is working in Firefox.

**Comment by nikitaorliak-cengage** [comment]
> > > > The panel opens with the new command on Safari and Firefox, but not on Chrome 🤔
> > > 
> > > 
> > > Sounds weird. I've tried using different keyboards and languages ​​in Chrome and Chromium and it works for me. Could you please double-check it?
> > 
> > 
> > I confirmed that on my PC, it works for all browsers (Chrome, Firefox and Edge). On my Mac, it does not work on Chrome but does on Firefox and Safari :/ @orion-cengage or @chris-cedrone-cengage , can you guys check if this works on your Macs?
> 
> Nope, the new keyboard command is not working for me in Chrome on my Mac, but it is working in Firefox.

Yeap, I tried a different OS, and only Chrome on Macs has this problem. I think it happens because Mac has his own shortcuts in Chrome and doesn't allow us to use ours :/

**Review by silvalaura — APPROVED** [review]
Looks good. Lets just update the PR title and description before merging

**Comment by silvalaura** [comment]
Lets hold off on merging for another day, as I will be releasing our next version first. I will merge this when we're ready.

**Review by orion-cengage — APPROVED** [review]
Update looks good to me. I confirmed the shortcut is gone.

**Pull request merged**
